src/test/java/com/ays/admin_user/service/impl/AdminUserServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package com.ays.admin_user.service.impl;
+
+import com.ays.AbstractUnitTest;
+import com.ays.admin_user.model.AdminUser;
+import com.ays.admin_user.model.dto.request.AdminUserListRequest;
+import com.ays.admin_user.model.dto.request.AdminUserListRequestBuilder;
+import com.ays.admin_user.model.entity.AdminUserEntity;
+import com.ays.admin_user.model.entity.AdminUserEntityBuilder;
+import com.ays.admin_user.model.mapper.AdminUserEntityToAdminUserMapper;
+import com.ays.admin_user.repository.AdminUserRepository;
+import com.ays.auth.model.AysIdentity;
+import com.ays.auth.model.enums.AysUserType;
+import com.ays.common.model.AysPage;
+import com.ays.common.model.AysPageBuilder;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.domain.Specification;
+
+import java.util.Collections;
+import java.util.List;
+
+class AdminUserServiceImplTest extends AbstractUnitTest {
+
+    @Mock
+    private AdminUserRepository adminUserRepository;
+
+    @Mock
+    private AysIdentity aysIdentity;
+
+    @InjectMocks
+    private AdminUserServiceImpl adminUserService;
+
+    private static final AdminUserEntityToAdminUserMapper ADMIN_ENTITY_TO_ADMIN_MAPPER = AdminUserEntityToAdminUserMapper.initialize();
```

Comment:
If we update the code to have two spaces between the Mapper objects and the objects defined as `@Mock`, it will improve readability.